packages/app/cypress/e2e/runs.cy.ts

```diff
@@ -171,6 +171,44 @@ describe('App: Runs', { viewportWidth: 1200 }, () => {
     })
   })
 
+  context('Runs - Create Project', () => {
```

Comment:
Not specific to your changes, but with name of this test file I wasn't expecting these tests to be specific to the dashboard.

Reply:
I see that some of the specs in this directory have the `cypress-in-cypress` prefix, i.e. `cypress-in-cypress-run-mode.cy.ts`. Should I update to something like `cypress-in-cypress-open-mode-runs.cy.ts`?

Reply:
We should probably retired the "cypress-in-cypress" convention, it refers to solving a specific problem, which was getting the spec runner to work in Cypress tests.

I do think "runs" is an OK name, since the the app contexts, runs and the "runs page" are all about test runs recorded to the dashboard. But `runs_page` is probably clearer file name if we want one, to be more clear that this is checking the display of runs info from the cloud, not anything at all about the behavior of the runner.

I even got confused writing that so we probably do need a better name.

Not that there are getting to be more files in here, it might be a good time to step back and organize this folder with more of pattern... some files test pages, some test multi-page features like the top-nav, and the names of the cypress-in-cypress tests are quite unclear.

Reply:
> I do think "runs" is an OK name, since the the app contexts, runs and the "runs page" are all about test runs recorded to the dashboard.

Given I haven't worked in the dashbaord, I alway associate runs to a spec run or test run within a spec. Def an overloaded term.!